sockeye/arguments.py

```diff
@@ -700,6 +700,10 @@ def add_training_args(params):
                               default=1.0,
                               help='Clip absolute gradients values greater than this value. '
                                    'Set to negative to disable. Default: %(default)s.')
+    train_params.add_argument('--gradient-clipping-type',
```

Comment:
can we rename to --clip-gradient-type to be consistent with the argument above?

Reply:
mh sure. or rename the other arg to `--gradient-clipping-treshold`, that even how it's called in train.py already.